tests/containers_test.go

```diff
@@ -406,6 +406,15 @@ func TestEnvironment(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	// All images must include git. Deployments uses this to checkout the source code.
+	t.Run("Git", func(t *testing.T) {
+		t.Parallel()
+
+		cmd := exec.Command("git", "--version")
+		_, err := cmd.Output()
```

Comment:
```suggestion
		err := cmd.Run()
```

should do the same thing I think?